server/index.js

```diff
@@ -40,5 +40,16 @@ app.listen(process.env.PORT, () => logger.info(`Listening on port ${process.env.
 
 process.on('SIGINT', () => {
   logger.info('Process received SIGINT: shutting down')
+  timeSaver.clearTime();
+  process.exit(1)
```

Comment:
change to exit(130). The practice when exiting cause of system signal is to exit with error code `128 + n` where `n` is the signal number.

Reply:
Done